### PR TITLE
Add schema.org metadata support

### PR DESCRIPTION
**Pull request by alebabai**
## what
* schema.org metadata support
* robots.txt

## why
#52 

## todo
- [x] use dict with json pipeline
- [x] breadcrumbs https://jsonld.com/breadcrumb/

### CODE REVIEW

**Comment by alebabai** [comment]
Useful links

- https://gohugo.io/content-management/urls/#canonicalization
- https://gohugo.io/templates/robots/
- https://keithpblog.org/post/hugo-website-seo/

File: layouts/robots.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+User-agent: *
```
Comment (osterman) [inline layouts/robots.txt]: This overwrites https://github.com/cloudposse/docs/blob/master/static/robots.txt

**Comment by osterman** [comment]
https://gohugo-amp.gohugohq.com/schema/breadcrumb/

File: layouts/partials/custom-head.html [hunk 2]
```diff
@@ -27,3 +27,4 @@
 <!-- Asciinema Player -->
 <link rel="stylesheet" type="text/css" href="/css/asciinema-player.css" />
 
+{{ partial "seo_schema" . }}
```
Comment (osterman) [inline layouts/partials/custom-head.html]: lets rename to `schema_org`

**Comment by osterman** [comment]
This is the inspiration for this PR: https://developers.google.com/search/docs/guides/search-gallery

**Pull request merged**